tests/appsec/iast_packages/test_packages.py

```diff
@@ -36,6 +36,58 @@
 # Remember to set to False before pushing it!
 _DEBUG_MODE = False
 
+IN_GITLAB = os.environ.get("GITLAB_CI", "false") in ("1", "true", "True")
+TEMPLATE_VENV_DIR = os.path.join(DDTRACE_PATH, "template_venv")
+CLONED_VENVS_DIR = os.path.join(DDTRACE_PATH, "cloned_venvs")
+PIP_EXECUTABLE = os.path.join(TEMPLATE_VENV_DIR, "bin", "pip")
+PIP_CACHE_SHARED_VENVS_DIR = os.path.join(DDTRACE_PATH, "pip_cache_shared_venvs")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cleanup(request):
+    """Cleanup the venv and cloned venvs directories at the start and end of the test session"""
+    # Clean up at the start
+    if os.path.exists(TEMPLATE_VENV_DIR):
```

Comment:
If I understand correctly, we should check the `DEBUG_MODE` variable here to avoid deleting the virtualenv, right?

Reply:
Yes. Initially I removed it with the hope of making this so fast debug mode wouldn't be needed, but it's only 30% faster at the end so I restored and the change will be in the next push.